### PR TITLE
Upload file

### PR DESCRIPTION
**Pull request by Avetiq30**
https://www.notion.so/B-Upload-File-Api-2bb7ce50a13e422ab5e811b5be611335?pvs=4
There is also a change that contradicts the pull request standard, but they are minor and will play a role in the work of the Api

### CODE REVIEW

File: src/file/file.service.ts [hunk 1]
```diff
@@ -10,9 +11,28 @@ export class FileService {
...
+
+  getFileExtension(filename: string): string {
+    const parts = filename.split('.');
+    return parts[parts.length - 1].toLowerCase();
+  }
+
   async uploadFile(file) {
-    const newFile = new this.fileModel(file);
-    return newFile.save();
+    const files = new this.fileModel(file);
```
Comment (VahagShahinyan) [inline src/file/file.service.ts:26]: what to do is this line ?

File: src/file/file.service.ts [hunk 2]
```diff
@@ -33,4 +53,8 @@ export class FileService {
     }
```
Comment (VahagShahinyan) [inline src/file/file.service.ts:53]: ```const deletedFile = this.fileModel.findByIdAndDelete(id); ```
    
    
    seems to be missing here await

File: src/file/file.service.ts [hunk 3]
```diff
@@ -10,9 +11,28 @@ export class FileService {
...
+  getFileExtension(filename: string): string {
+    const parts = filename.split('.');
+    return parts[parts.length - 1].toLowerCase();
+  }
+
   async uploadFile(file) {
-    const newFile = new this.fileModel(file);
-    return newFile.save();
+    const files = new this.fileModel(file);
+    if (!files) {
```
Comment (VahagShahinyan) [inline src/file/file.service.ts:27]: delete this check

File: src/file/file.service.ts (hunk 2 above)
Comment (Avetiq30) [reply src/file/file.service.ts:53]: ++

File: src/file/file.service.ts (hunk 1 above)
Comment (Avetiq30) [reply src/file/file.service.ts:26]: ++

File: src/file/file.service.ts (hunk 3 above)
Comment (Avetiq30) [reply src/file/file.service.ts:27]: ++

**Review by VahagShahinyan — APPROVED** [review]

**Pull request merged**